src/components/Drawer/index.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { ReactElement } from 'react';
+import clsx from 'clsx';
+import { Flipped, Flipper } from 'react-flip-toolkit';
+import DrawerContext from './context';
+import { Props } from './types';
+
+
+/**
+ * The Drawer component is used when you want to have content slide out from either the Top, Right, Bottom, or Left
+ */
+function Drawer({
+    anchorFrom,
+    children,
+    className,
+    isOpen = false,
+    ...otherProps
+}: Props): ReactElement {
+
+
+    const [open, setOpen] = React.useState(isOpen);
+
+
+    React.useEffect(() => {
+        setOpen(isOpen);
+    }, [isOpen]);
+
```

Comment:
I'm not sure this is necessary, I had to add this for the isOpen knob to work in Storybook...but that doesn't really matter in real practice

```
const isOpen = boolean('isOpen', false); <---- this wouldn't trigger rerenders without the React.useEffect() call

return <Drawer isOpen={isOpen}> content </Drawer>
```

Reply:
You might be able to get around the whole rerender thing by splitting out your context provider from the Drawer component.  Just a thought...